sql/moz-fx-data-shared-prod/fenix_derived/feature_usage_events_v1/query.sql

```diff
@@ -0,0 +1,1461 @@
+/*History*/
+  history_opened,
+  history_opened_users,
+  history_opened_item,
+  history_opened_item_users,
+  history_opened_items_in_new_tabs,
+  history_opened_items_in_new_tabs_users,
+  history_opened_items_in_private_tabs,
+  history_opened_items_in_private_tabs_users,
+  history_recent_searches_tapped,
+  history_recent_searches_tapped_users,
+  history_remove_prompt_cancelled,
+  history_remove_prompt_cancelled_users,
+  history_remove_prompt_opened,
+  history_remove_prompt_opened_users,
+  history_removed,
+  history_removed_users,
+  history_removed_all,
+  history_removed_all_users,
+  history_removed_last_hour,
+  history_removed_last_hour_users,
+  history_removed_today_and_yesterday,
+  history_removed_today_and_yesterday_users,
+  history_search_icon_tapped,
+  history_search_icon_tapped_users,
+  history_search_term_group_open_tab,
+  history_search_term_group_open_tab_users,
+  history_search_term_group_remove_all,
+  history_search_term_group_remove_all_users,
+  history_search_term_group_remove_tab,
+  history_search_term_group_remove_tab_users,
+  history_search_term_group_tapped,
+  history_search_term_group_tapped_users,
+  history_shared,
+  history_shared_users,
+/*FxA*/
+  sync_failed,
+  sync_failed_users,
+  sync_account_opened,
+  sync_account_opened_users,
+  sync_account_send_tab,
+  sync_account_send_tab_users,
+  sync_account_sign_in_to_send_tab,
+  sync_account_sign_in_to_send_tab_users,
+  sync_account_sync_now,
+  sync_account_sync_now_users,
+  sync_auth_closed,
+  sync_auth_closed_users,
+  sync_auth_opened,
+  sync_auth_opened_users,
+  sync_auth_other_external,
+  sync_auth_other_external_users,
+  sync_auth_paired,
+  sync_auth_paired_users,
+  sync_auth_recovered,
+  sync_auth_recovered_users,
+  sync_auth_scan_pairing,
+  sync_auth_scan_pairing_users,
+  sync_auth_sign_in,
+  sync_auth_sign_in_users,
+  sync_auth_sign_out,
+  sync_auth_sign_out_users,
+  sync_auth_sign_up,
+  sync_auth_sign_up_users,
+  sync_auth_use_email,
+  sync_auth_use_email_users,
+  sync_auth_use_email_problem,
+  sync_auth_use_email_problem_users,
+/*Privacy*/
+  hp_private_mode_tapped,
+  hp_private_mode_tapped_users,
+  tab_tray_private_mode_switched,
+  tab_tray_private_mode_switched_users,
+  app_icon_private_tab_tapped,
+  app_icon_private_tab_tapped_users,
+  tab_tray_private_mode_tapped,
+  tab_tray_private_mode_tapped_users,
+  etp_setting_changed,
+  etp_setting_changed_users,
+  etp_settings,
+  etp_settings_users,
+  etp_shield,
+  etp_shield_users,
+  etp_tracker_list,
+  etp_tracker_list_users,
+/*Default Browser*/
+  default_browser_changed_users,
+  default_browser_changed,
+/*Notification*/
+  re_engagement_notif_shown,
+  re_engagement_notif_shown_users,
+  re_engagement_notif_tapped,
+  re_engagement_notif_tapped_users,
+/*Customize Home*/
+  app_menu_customize_homepage,
+  app_menu_customize_homepage_users,
+  home_page_customize_home_clicked,
+  home_page_customize_home_clicked_users
+FROM
+  dau_segments d
```

Comment:
```suggestion
  dau_segments AS segments
```

Same as below, please make the alias more meaningful, single letter aliases are not very helpful. And does not adhere to out SQL styling rules: https://docs.telemetry.mozilla.org/concepts/sql_style.html#aliasing